src/Controls/src/Core/Compatibility/Handlers/Shell/iOS/ShellSearchResultsRenderer.cs

```diff
@@ -37,7 +38,22 @@ DataTemplate DefaultTemplate
 				return _defaultTemplate ?? (_defaultTemplate = new DataTemplate(() =>
 					{
 						var label = new Label();
-						label.SetBinding(Label.TextProperty, SearchHandler.DisplayMemberName ?? ".");
+
+						if (RuntimeFeature.IsShellSearchResultsRendererDisplayMemberNameSupported)
+						{
+							label.SetBinding(Label.TextProperty, SearchHandler.DisplayMemberName ?? ".");
+						}
+						else
+						{
+							if (SearchHandler.DisplayMemberName is not null)
```

Comment:
Let's try adding `[Obsolete]` and see if we get XAML warnings when you use it.

Reply:
So XamlC will produce warnings for `[Obsolete]` properties:
https://github.com/dotnet/maui/blob/37ca0c26e39a40f14b4fd60416b90a1a62e28473/src/Controls/src/Build.Tasks/SetPropertiesVisitor.cs#L1027-L1029

Now the question is if we really want to make it `[Obsolete]` and force even customers who don't want to use NativeAOT or enable trimming to use `ItemTemplate` instead.

Reply:
I would think this one property isn't widely used. I would say just `[Obsolete]` completely, and we can see if there are any complains in early .NET 9 previews?